CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.2] - 2021-05-05
+
+### Added
```

Comment:
- [ ] Please move under `[Unreleased]` and do not introduce a new version

(FYI, all those updates regarding a new version are part of the release process. no need to do them in a PR)